platform/ios/src/MGLMapView.mm

```diff
@@ -387,10 +387,9 @@ - (void)commonInit
     self.accessibilityTraits = UIAccessibilityTraitAllowsDirectInteraction | UIAccessibilityTraitAdjustable;
     _accessibilityCompassFormatter = [[MGLCompassDirectionFormatter alloc] init];
     _accessibilityCompassFormatter.unitStyle = NSFormattingUnitStyleLong;
-
     self.backgroundColor = [UIColor clearColor];
     self.clipsToBounds = YES;
-
+    if (@available(iOS 11.0, *)) { self.accessibilityIgnoresInvertColors = YES; }
```

Comment:
As in https://github.com/mapbox/mapbox-gl-native/pull/9876#issuecomment-325739596, using `@available` means that we’re requiring Xcode 9 to build this SDK. If we choose to require Xcode 9, there are some further steps we should take: #10452